engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -608,13 +609,16 @@ protected void serviceStart() throws Exception
 
     try {
       Configuration config = getConfig();
+      YarnConfiguration conf = new YarnConfiguration();
```

Comment:
Why is it necessary to have Configuration and YarnConfiguration? Can this code be simplified to `YarnConfiguration = StramClientUtils.getYarnConfiguration(getConfig());`